apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -150,6 +151,47 @@ Page {
                 }
                 onClicked: pageStack.push("PermissionPage.qml")
             }
+
+            BrowserListItem {
+                //% "Save destination"
```

Comment:
It may be that this is part of a fixed design already, but for me "Download destination" would be clearer than "Save destination".

Reply:
Now that I've seen the design, please ignore this comment; this is the correct text following the design.